packages/foam-core/src/model/workspace.ts

```diff
@@ -221,12 +223,12 @@ export class FoamWorkspace implements IDisposable {
     ];
   }
 
-  public static getLinks(workspace: FoamWorkspace, uri: URI): URI[] {
-    return workspace.links[uri.path]?.map(c => c.target) ?? [];
+  public static getLinks(workspace: FoamWorkspace, uri: URI): Connection[] {
```

Comment:
to support multiple links across notes, `getLinks` and `getBacklinks` no longer return the `URI`, but the full `Connection` object.
An alternative would have been for these to just return the unique URIs, and leave it to `getAllConnections` to return the fully detailed list of connections. 
I am open to POVs here